flambe/field/text.py

```diff
@@ -258,7 +325,7 @@ def _build_embeddings(self, model: KeyedVectors) -> Tuple[odict, torch.Tensor]:
 
         tokens: Iterable[str] = self.vocab.keys()
 
-        if self.setup_all_embeddings:
+        if setup_all_embeddings:
```

Comment:
I would rename to avoid the outer scope naming collapse. It would become clearer it is definitely a local variable